<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Excel Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Settings">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Set up</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="2" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Sheet:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Sheets"/>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="ChooseButton">
             <property name="text">
              <string>Choose File</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Select Excel table to open:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1" colspan="2" alignment="Qt::AlignTop">
            <widget class="QLabel" name="Filename">
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Choose row number from</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="Row_start">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Choose column range from</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="Col_start">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="Col_end">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="Row_end">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="GenerateButton">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Generate PDFs</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="Output_message">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>0 PDFs will be generated</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Data">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>358</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="Displayed_data">
             <property name="text">
              <string>Data here</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
